### DISCUSSION TITLE
Neo Electronics - PIR - NAS-PD07 - Unknown product 0x0720

### DISCUSSION THREAD

**Discussion by ngblume** [zwave-js/zwave-js-ui#1703] [Q&A] [answered]
Hello everyone,

I'm not sure, if this is the right place to ask this question..
If not, please direct me to where to ask my question...

I purchased a NEO Z-Wave Motion Sensor - NAS-PD07 - SKUH74049 (EU).
It got included easily, but is reported as "Unknown product 0x0720".
![image](https://user-images.githubusercontent.com/3869392/133926877-27154113-0ed1-4226-9901-4484a3c236b7.png)
And I think therefore most of the parameters mentioned in the manual are missing for the WebUI...

When I look at the screenshots (https://zwave-js.github.io/zwavejs2mqtt/#/screenshots?id=control-panel), "Control-Panel" shows a lot of NEO devices, but not the NAS-PD07.
Where can I find a list of all currently supported devices to check whether a device is supported or not?

Assuming that this devices is currently not supported, I looked into custom devices and found this in the docs:
https://zwave-js.github.io/zwavejs2mqtt/#/usage/custom-device-files
This mentions "<app-root-folder>/store/config" as the path for such a file..
Unfortunately, I'm not able to find this path in my Z-WaveJS2MQTT installation in Home Assistant.
Can someone please point me in the right direction?

Where can I find the template / style guide for device files or can I directly use an Open-Z-Wave file?
(i.e. https://github.com/OpenZWave/open-zwave/tree/master/config/shenzen_neo, but there is no file for NAS-PD07 as well)

Thanks!

Cheers
Niels Göran

**Comment by kpine** [comment, marked as answer]
> Unfortunately, I'm not able to find this path in my Z-WaveJS2MQTT installation in Home Assistant.

Go to the Store Explorer in the zwavejs2mqtt UI and create a `config` folder. Then you can navigate to that folder and upload a file.

**Reply by ngblume** [reply]
Thanks, found that as well but wasn't sure if this would create the correct path.
